system/ee/legacy/libraries/Template.php

```diff
@@ -1889,7 +1914,7 @@ public function fetch_cache_file($cfile, $cache_type = 'tag', $args = array())
         $status = & $this->$status;
 
         // Bail out if this tag/template isn't set to cache
-        if (ee('LivePreview')->hasEntryData() or ! isset($args['cache']) or $args['cache'] != 'yes') {
+        if (! isset($args['cache']) or $args['cache'] != 'yes') {
```

Comment:
what is the reason check for live preview was taken out?

Reply:
Was not an intended change - perhaps it is a sync issue and I was working from an old version of code?

Reply:
On checking the code you refer too (`if (ee('LivePreview')->hasEntryData() or ! isset($args['cache']) or $args['cache'] != 'yes') {` does not appear to be part of code in branch 6.dev upon which this edit was made.  What would you like done?